snapshots/smoking/2024-05-30/cigarette_sales.zip.dvc

```diff
@@ -0,0 +1,28 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: International Smoking Statistics
+    date_published: "2016-12-19"
+    title_snapshot: International Smoking Statistics - Cigarette Sales
+
+    # Citation
+    producer: Barbara Forey, Jan Hamling, John Hamling, Alison Thornton, Peter Lee
+    attribution: International Smoking Statistics (2016)
+    citation_full: |-
+      International Smoking Statistics WEB Edition
+      © Barbara Forey, Jan Hamling, John Hamling, Alison Thornton, Peter Lee, 2006-2016
```

Comment:
Do they explicitly ask to use this citation? If yes, then it's fine (we should follow the data providers' recommendations). If not, I think the copyright does not belong here. We have a field for license below (although it's currently not surfaced in data pages, but it should be at some point).
In this case, the full citation should mention the producer (as well as the authors) and the title of the data product.
So, I'd suggest:
P N Lee Statistics and Computing Ltd - International Smoking Statistics WEB Edition.
Barbara Forey, Jan Hamling, John Hamling, Alison Thornton, Peter Lee (2006-2016).